client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1075,6 +1090,11 @@ EndpointBuilder addCommaSeparatedPathParts(String[] parts) {
             return this;
         }
 
+        EndpointBuilder addCommaSeparatedPathParts(List<String> parts) {
```

Comment:
https://github.com/elastic/elasticsearch/blob/master/client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java#L966

Reply:
I just added this to save a `toArray()` call here, seemed cleaner to just add that API, but I don't have a strong opinion here :)